modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -122,9 +127,26 @@ public String getRESTContextPath() {
 			throw new UnsupportedOperationException();
 		}
 
+		ObjectDefinition rootObjectDefinition =
+			_objectDefinitionPersistence.fetchByPrimaryKey(
```

Comment:
In the master's branch, I see this code to do the same action:

```
ObjectDefinition rootObjectDefinition =
			ObjectDefinitionLocalServiceUtil.fetchObjectDefinition(
				getRootObjectDefinitionId());
```

Reply:
This is a similar pattern I found to mock the `ObjectDefinitionLocalServiceUtil`:

![image](https://github.com/liferay-headless/liferay-portal/assets/32699538/5f1e0772-9941-4d7b-8b4b-dd7f3059f072)